packages/zipkin-transport-kafka/README.md

```diff
@@ -0,0 +1,25 @@
+# Zipkin-transport-kafka
+
+This is a module that sends Zipkin trace data from zipkin-js to Kafka.
+
+## Usage:
+
+`npm install zipkin-transport-kafka --save`
+
+```javascript
+const {Tracer, BatchRecorder} = require('zipkin');
+const KafkaLogger = require('zipkin-transport-kafka');
+
+const kafkaRecorder = new BatchRecorder({
```

Comment:
Yeah, lingo needs to be consistant, and might also be nice with an little readme about the lingo :).